src/build-time-render/BuildTimeRender.ts

```diff
@@ -71,6 +72,7 @@ export interface BuildTimeRenderArguments {
 		invalidates?: string[];
 		excludes?: string[];
 	};
+	features?: FeatureMap | Features;
```

Comment:
I think we should accept the compiled map that would be passed by the build. So i don't think we need to use `getFeatures` just pass the features onwards.

Reply:
yeah, it would be nice if we could drop the dependency on `static-build-loader` completely (for types also).